examples/ami/ami.ipynb

```diff
@@ -0,0 +1,264 @@
+{
+ "cells": [
```

Comment:
The last cell rendered in this notebook doesn't look right - the VAD label says the first 1/3rd is speech and the rest is silence, whereas the spectrogram suggests something else. 

See: https://github.com/pzelasko/lhotse/blob/b55e349d40ed88ecb2ad411ba0002405af4845d0/examples/ami/ami.ipynb

Reply:
I have noticed this problem last week, and I double checked it. I think the codes of lhotse is correct,  but the time point labeling of AMI corpus is not very precise. Manually I modified the labels of an example for unit test: `test/fixtures/ami/cuts.yml`, and the result from the unit test looks correct.